pkg/rancher-desktop/main/snapshots/types.ts

```diff
@@ -3,6 +3,7 @@ export type SnapshotEvent = {
   result?: 'success' | 'cancel' | 'error',
   error?: string,
   snapshotName?: string,
+  creationTime?: string,
```

Comment:
I keep getting confused that this is the time the snapshot action was initiated, rather than completed. Would you mind calling it `eventTime` instead?

I should be able to be convinced otherwise, though :)